pylint/reporters/ureports/nodes.py

```diff
@@ -120,16 +125,23 @@ class Section(BaseLayout):
     as a first paragraph
     """
 
-    def __init__(self, title=None, description=None, **kwargs):
+    def __init__(
+        self,
+        title: Optional[str] = None,
+        description: Optional[str] = None,
+        **kwargs: Any,
+    ) -> None:
         super().__init__(**kwargs)
         if description:
             self.insert(0, Paragraph([Text(description)]))
         if title:
             self.insert(0, Title(children=(title,)))
+        # Used in ReportHandlerMixin make_reports
+        self.report_id = None
```

Comment:
https://github.com/PyCQA/pylint/pull/4973/ adds this to the `__ini__`, which might make this unneeded.

Reply:
I don't understand, report_id seems to still be used, right ?

Reply:
OK, there's still a little change because I chose str instead of optional[str],  but we did the same main change that was already handled in #4973.

Reply:
Yeah, I made this comment before the merge of #4973. You can leave whatever you prefer, I don't have a particular preference although I would perhaps add the comment on the same line.